app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>All Recipes</h1>
+
+<% @recipes.each do |recipe| %>
```

Comment:
Your instance variable that you set up in the view is called `@recipe` singular because it's just one recipe, so this probably gave you an error